common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -391,12 +403,29 @@ def get_progress(self):
                 return None
         return None
 
+    def show_progress(self, **kwargs):
+        """
+        Return (score, total) to be displayed to the learner.
+        """
+        if 'progress' in kwargs:
```

Comment:
> If we have to have a separate method, I would suggest `get_display_progress` as the method name.

For now, I think `get_display_progress` is sufficient.  8050bf304e renames the method, and removes the `kwargs`. 

> Actually, wouldn't it make more sense to just embed the logic of making the score None in `get_progress` itself? Is there a use case for wanting show_progress to return the actual score value when `self.correctness_available()` is False?

That's a good point, and something worth discussing for the follow-up PR, where we withhold correctness from the Progress page.

Currently, the `get_progress()` method is shared by all xmodules, and can return `None` or a [`Progress` object, which cannot have a `score` of `None`](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/progress.py#L11-L21).  Since we still want to be able to show the `total` points possible, but without the `score` in some cases, it seems worth discussing a modification to the `Progress` object that allows `score = None`, so that the [sub/section `get_progress()` calculations can use the same logic](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/seq_module.py#L184-L192).

@nasthagiri any thoughts?